change_logs/release_v0.20.0.md

```diff
@@ -16,7 +16,7 @@ On Slack? Please join us [K9slackers](https://join.slack.com/t/k9sers/shared_inv
 
 And now for something a `beat` different?
 
-I figured, why not share one of the tunes I was spinning when powering thru teh bugs? Might as well share the pain/pleasure roaght?
```

Comment:
If you're attached to these spellings, I can trivially drop individual commits.

Reply:
well this is for fun - usually write the release docs late at night so a little liberty is in order...

Reply:
Thanks Josh - inside humor likely no one gets - so do make it `right`!